### DISCUSSION TITLE
Porque meu Intellisence não Funfa ?

### DISCUSSION THREAD

**Discussion by cgomesSynctech** [balta-io/7196#13] [Perguntas e Respostas] [answered]
Não sei o que esta faltando no meu VS. Não sei porque não aparece as sugestões e implementações necessárias ao método, isso esta gerando hora erros e hora entendimento errado pelo VSCode. por exemplo . abaixo a foto da classe Contexts do projeto. 
mesmo colocando o using, o DbContext não aparece nas sugestões .. se eu selecionar a opção ( que teoricamente seria a ef-dbcontext) ele gera o codigo completo da implementação na classe .... não sei se consegui explicar o fato !!! minha cabeça esta doida com isso ! 

![image](https://user-images.githubusercontent.com/30666467/130848760-0d8a9c73-d55f-4045-b3c2-aaae0be6a06f.png)

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde @cgomesSynctech também não sei o que pode ser.
Tentaria remover o VS, remover o .NET e reinstalar tudo de novo...

**Reply by cgomesSynctech** [reply]
Bem .. Decide não reinstalar e tentar achar o problema !!!! 
ai fui "catucar", notei que na parte inferior ( o VS chama de barra de Status ) da tela ficava a informação de um dos projetos abertos, ai quando mudei o projetos _"voila"_ o intellisense passou a funcionar no projeto !!!!
![image](https://user-images.githubusercontent.com/30666467/131023728-30ea07bc-2926-463b-8f3f-e7dda3ce515f.png)

**Reply by cgomesSynctech** [reply]
@andrebaltieri .. na imagem colada na pergunta ate aparece o Todo.Dominio, ou seja o que estava selecionado !!!!
